.github/workflows/platform.set-avm-github-issue-label.yml

```diff
@@ -0,0 +1,41 @@
+# Workflow for checking issues if correct labels are applied and corrects if necessary
+name: .Platform - Set GitHub issue label
+
+on:
+  schedule:
```

Comment:
If the logic is the same I'd suggest a single platform pipeline, running on schedule as well as on issue creation (issues types [opened])

Reply:
@eriqua it is the same logic, but the issue creation one only looks into one issue (the new one), while the scheduled one looks at all open issues, so there is a loop into it, that calls the other logic

Reply:
My suggestion would be to merge the 2 and use `github.event_name` to check which event triggered the workflow. Based on that call the corresponding intended logic.

However, given another PR #4987 was opened meanwhile and updating some of the common logic, I'd suggest to have this PR in to allow the review of the other one. Then, think about the optimization later.